source/adios2/helper/adiosKokkos.cpp

```diff
@@ -43,6 +43,10 @@ void KokkosMinMaxImpl(const char * /*values*/, const size_t /*size*/,
                       char & /*min*/, char & /*max*/)
 {
 }
+void KokkosMinMaxImpl(const long double * /*values*/, const size_t /*size*/,
+                      long double & /*min*/, long double & /*max*/)
+{
+}
```

Comment:
You could possibly use `Kokkos::abort` for all of these.

Reply:
I am not aborting because for now I don't want to terminate the program, just not compute the metadata on the GPU.